examples/pytorch/text-generator/README.md

```diff
@@ -48,36 +41,41 @@ torch
 transformers==3.0.*
 ```
 
-<br>
-
-## Configure your API
+## Deploy your model locally
 
-Create a `cortex.yaml` file and add the configuration below. A `RealtimeAPI` provides a runtime for inference and makes your `predictor.py` implementation available as a web service that can serve realtime predictions:
+You can create APIs from any Python runtime (e.g. the Python shell or a Jupyter notebook):
 
-```yaml
-# cortex.yaml
+```python
+import cortex
 
-- name: text-generator
-  kind: RealtimeAPI
-  predictor:
-    type: python
-    path: predictor.py
-```
+cx = cortex.client("local")
 
-Here are the complete [API configuration docs](../../../docs/deployments/realtime-api/api-configuration.md).
+api_spec = {
+  "name": "text-generator",
+  "kind": "RealtimeAPI",
+  "predictor": {
+    "type": "python",
+    "path": "predictor.py"
+  }
+}
 
-<br>
+cx.deploy(api_spec, project_dir=".", wait=True)
+```
 
-## Deploy your model locally
+## Consume your API
 
-`cortex deploy` takes your Predictor implementation along with the configuration from `cortex.yaml` and creates a web API:
+```python
+import requests
 
-```bash
-$ cortex deploy
+endpoint = cx.get_api("text-generator")["endpoint"]
+payload = {"text": "hello world"}
+print(requests.post(endpoint, payload).text)
 
-creating text-generator (RealtimeAPI)
+# cx.delete_api("text-generator")
```

Comment:
Consider moving this to the end, in a "cleanup" section (in there, you might mention that you can delete it in python or from the CLI)